addons/binding/org.openhab.binding.insteonplm/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.openhab.binding.insteonplm</name>
```

Comment:
Please don't add insteonplm files to this PR.

Reply:
Looks like I missed some problems in the first few commits. Rebased and made one commit for the initial code by @pinkfish, and then added my changes on top.